tools/topology/topology2/cavs/platform/intel/cnl.conf

```diff
@@ -0,0 +1,5 @@
+# CNL/CML/CFL-specific variable definitions
+Define {
+	DMIC_IO_CLK		24000000
+	DMIC_DRIVER_VERSION	1
```

Comment:
DRIVER_VERSION?

Is this hardware version maybe? What does this refer to?

Reply:
@juimonen can you please comment?

Reply:
The DMIC driver supports 3 versions of DMIC.

Reply:
yes dmic hw version